autode/transition_states/ts_guess.py

```diff
@@ -30,35 +31,42 @@ def get_ts_guess_constrained_opt(reactant, method, keywords, name, distance_cons
 
     mol_with_const = deepcopy(reactant)
 
-    # Run a low level constrained optimisation first to prevent the DFT being problematic if there are >1 constraint
+    # Run a low level constrained optimisation first to prevent the DFT being
+    # problematic if there are >1 constraint
     l_method = get_lmethod()
-    ll_const_opt = Calculation(name=f'{name}_constrained_opt_ll', molecule=mol_with_const, method=l_method,
-                               keywords_list=l_method.keywords.low_opt, n_cores=Config.n_cores, opt=True,
-                               distance_constraints=distance_consts, point_charges=get_species_point_charges(mol_with_const))
+    ll_const_opt = Calculation(name=f'{name}_constrained_opt_ll',
+                               molecule=mol_with_const, method=l_method,
+                               keywords=l_method.keywords.low_opt,
+                               n_cores=Config.n_cores,
+                               distance_constraints=distance_consts)
     ll_const_opt.run()
 
-    # Try and set the atoms, but continue if they're not found as hopefully the other method will be fine(?)
+    # Try and set the atoms, but continue if they're not found as hopefully the
+    # other method will be fine(?)
     try:
         mol_with_const.set_atoms(atoms=ll_const_opt.get_final_atoms())
 
     except AtomsNotFound:
         pass
 
-    hl_const_opt = Calculation(name=f'{name}_constrained_opt', molecule=mol_with_const, method=method, opt=True,
-                               keywords_list=keywords, n_cores=Config.n_cores, distance_constraints=distance_consts,
-                               point_charges=get_species_point_charges(mol_with_const))
-    hl_const_opt.run()
+    hl_const_opt = Calculation(name=f'{name}_constrained_opt',
+                               molecule=mol_with_const, method=method,
+                               keywords=keywords, n_cores=Config.n_cores,
+                               distance_constraints=distance_consts)
 
-    # Form a transition state guess from the optimised atoms and set the corresponding energy
+    # Form a transition state guess from the optimised atoms and set the
+    # corresponding energy
     try:
-        mol_with_const.run_const_opt(hl_const_opt, method, Config.n_cores)
+        mol_with_const.run_const_opt(hl_const_opt)
     except AtomsNotFound:
+        # Retrun with the low level
         try:
-            mol_with_const.run_const_opt(ll_const_opt, get_lmethod(), Config.n_cores)
+            mol_with_const.run_const_opt(ll_const_opt)
```

Comment:
this isn't needed any more without explicit solvent, I don't think

Reply:
I agree, but will be when it's reintroduced?

Reply:
I guess, but I feel like we should still remove it until then

Reply:
Sure. Do we need the energy from the calculation?

Reply:
nope